README.md

```diff
@@ -26,15 +26,26 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
   ```
 - Build the binary file.
   ```
-  go build -i -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
+  go build -o $GOPATH/bin/nimble-gin github.com/nimblehq/gin-templates
   ```
-- Run create the project command.
+- Create the project from a template on a specifying branch.
   ```
-  nimble-gin create
+  nimble-gin -b {{git branch}} create
   ```
+  **Note:** `-b` can be omitted, branch will be `main` by default.
```

Comment:
I recommend keeping the general use case by default in the README. Then in the wiki, there can be additional options detailed. So I recommend to revert this change and add wiki page to explain all options.

Reply:
Fixed in 2a50e67df6f4112a5236c6eba238a736dacc2df9.
I created a new issue to add command's flags information in https://github.com/nimblehq/gin-templates/issues/26